tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>I've said this on email, but this won't work since clustering and pruning don't create new weight objects.</p><p> </p><p>So when you are training clustered_model, you also modify the weights of model.</p><p> </p><p>For now, I've kept them separate (see <a href="https://github.com/tensorflow/model-optimization/pull/388" target="_blank">https://github.com/tensorflow/model-optimization/pull/388</a>), though we should update it to have independent weight objects, like QAT does. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>